src/app/zones/views/ZoneTabsView.vue

```diff
@@ -1,11 +1,33 @@
+<script lang="ts" setup>
+import AppView from '@/app/application/components/app-view/AppView.vue'
+import RouteView from '@/app/application/components/route-view/RouteView.vue'
+import NavTabs from '@/app/common/NavTabs.vue'
+</script>
```

Comment:
**Comment**: I really don’t think we spent our time well by moving the script section to the top of files. In a component, the template should have the priority as that’s what we’re building, that’s the result. What a component imports is at best secondary and can be looked up easily enough.

Reply:
We discussed this offline, as far as I remember the decision was to move the headers to the top of the file throughout the project.

---

The above decision was based on the fact that all that is left in the majority of files now are the JS/TS 'headers', and they should be at the 'head' of the file. In the past I've found it really weird having to find the headers in the middle of the file and having to guess where they are to take a quick look at them instead of jumping to the 'head' of the file. 

>  I really don’t think we spent our time well by moving the script section

From what I remember I think in our previous iteration of the project you spent a lot of time moving these around, I think you mentioned this in a PR a while back. The project has vastly changed since then and as mentioned above, pretty much all that is left in terms of JS/TS are the 'headers'

There is a tool to do this plus an eslint rule for the future, I plan to PR that change soon, (also see https://twitter.com/youyuxi/status/1427982990957846529) so comparatively there will far less time spent moving these to the 'head' of the file.

Let me know if you still have any concerns.

Reply:
We've now spent more effort discussing this than any other option. This is really getting into a tabs vs space argument here.